wildfly/resteasy-manifest/pom.xml

```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~
+  ~ Copyright 2023 Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-jaxrs-all</artifactId>
+        <version>7.0.0.Alpha2-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>resteasy-manifest-7.0</artifactId>
```

Comment:
Is this version in the `artifactId` mandatory for the manifest to work? I mean for each release, do we need to maintain a different version of manifest like this?

Reply:
The version makes it possible to create a separate manifest stream for different releases. This way channels can detect the latest manifest in each stream to e.g. perform update.

I wonder if it would be possible to create a property in top level pom and use it in other poms so it's easier to manage.

Reply:
@spyrkob I had it working like that, e.g. a dynamic `artifactId`. The problem was you have to override the `maven-deploy-plugin` and the `maven-install-plugin` to skip the default `deploy` and `install` goals. You then need an execution for the `deploy-file` and `install-file` goals. While those seem to work, they seem fragile and could break other plugins.

One option I thought of is to keep the artifact ID generic, e.g. `resteasy-manifest` and create an additional manifest for the release stream and have the one with the version left off always be the latest.